### ISSUE TITLE
KappaPT - P18a - Entity objects change color on pickup

### ISSUE THREAD

**Issue by dxfuzion** [rre36/ShaderpackIssueTracker#98]
**Describe the bug**
When a player nears an entity object to pick it up, the object takes on a dark appearance with a random colorful tint. It appears to be some kind of world space normal buffer.

**To Reproduce**
1. Start a new Minecraft 1.20.1 project
2. Add Fabric 15.7, Iris Shaders 1.6.17, Sodium 0.5.8 (current latest versions)
3. Load into a game and enable KappaPT shader
4. Drop or break a block
5. Observe that entity blocks change appearance when being picked up

**Expected behavior**
Entity block maintains consistent appearance when being picked up. 

**Screenshots**
![image](https://github.com/rre36/ShaderpackIssueTracker/assets/152972820/8f6d4378-7d2f-468a-b085-142a2a39e90b)

https://github.com/rre36/ShaderpackIssueTracker/assets/152972820/8fef70bd-2d2d-4d38-b990-5c58689c521d

**Environment (please complete the following information):**
Iris 1.6.17
Minecraft Version: 1.20.1
Shaderpack Version:KappaPT - P18a 
Graphics Card and Driver Version: 4090 RTX, 551.61
Operating System: Win 11 Pro
CPU: i9 13900k
Amount of System Memory (total and assigned to Java): 64gb 

**Additional context**
Was instructed to use Iris with this shader when trying to load it via Optifine.

**Comment by rre36** [comment]
Thats just a weird minecraft rendering quirk on dropped objects, nothing that can really be done about that.

**Comment by dxfuzion** [comment]
@rre36  Hello! Thank you for your reply. I believe this is not the case. I have some additional context that may help.

I would like to point out I have only observed the bug on Kappa shaders, it occurs on _all_ Kappa shaders. It is a bug _exclusive_ to your shader packs. No other developers shader I have used has this bug, including other path tracing and voxel based shaders. 

Based on this information, I believe is is reasonable to think the bug is within the Kappa shader packs, something about your method used across all Kappa shaders, not the Minecraft game code.

I hope you consider reevaluating this ticket. Fixing this bug would be a great boon to all your packs, it is an eyesore that detracts from gameplay, your shaders are my favorite visually by far! But sadly, I have been forced to move on to other shader author's that don't have this bug.

**Comment by rre36** [comment]
Trust me, it's exactly the same rendering quirk that has been an issue for ages and it plagues every single shaderpack that isn't doing forward shading. It's just especially prominent with my packs because I am one of the few people to actually use a very wide range of brightness values in most of my packs.
Tl;dr of what happens there, the dropped items change their rendering state when being picked up, but aren't rendered in the matching, different order in the rendering pipeline. It's been a thing since 1.15 or 1.16 I think.
